apps/etterlatte-saksbehandling-ui/client/src/shared/types/Aktivitetsplikt.ts

```diff
@@ -108,13 +109,34 @@ export interface IOpprettAktivitetspliktAktivitetsgrad {
   id: string | undefined
   aktivitetsgrad: AktivitetspliktVurderingType
   fom: string
+  tom?: string
   beskrivelse: string
 }
 
 export interface IOpprettAktivitetspliktUnntak {
   id: string | undefined
   unntak: AktivitetspliktUnntakType
-  fom?: string
+  fom: string
   tom?: string
   beskrivelse: string
 }
+
+export interface AktivitetspliktVurderingValues {
+  aktivitetsplikt: JaNei | null
+  aktivitetsgrad: AktivitetspliktVurderingType | ''
+  unntak: JaNei | null
+  midlertidigUnntak: AktivitetspliktUnntakType | ''
+  fom?: Date | null
+  tom?: Date | null
+  beskrivelse: string
```

Comment:
bytte ut `null` som type med `undefined`?

Reply:
Bruker null for å unngå uncontrolled rhf components